patterns/travel-information.php

```
@@ -0,0 +1,179 @@
+);
+
+/**
+ * Generate card markup for a single travel information card
+ *
+ * @param array $card Card configuration array.
+ * @return string Card HTML markup.
+ */
+function lsx_to_generate_travel_info_card( $card ) {
+	$slug          = $card['slug'];
+	$title         = $card['title'];
+	$meta_key      = $card['meta_key'];
```

PHPMD (2.15.0)

The variable $meta_key is not named in camelCase. (undefined)

(CamelCaseVariableName)